staging/src/k8s.io/kubectl/pkg/cmd/create/create_pdb_test.go

```diff
@@ -194,16 +194,16 @@ func TestCreatePdb(t *testing.T) {
 					Name: "my-pdb",
 				},
 				Spec: policyv1.PodDisruptionBudgetSpec{
-					Selector:     selector,
-					MinAvailable: &minUnavailableNumber,
+					Selector:       selector,
+					MaxUnavailable: &minUnavailableNumber,
```

Comment:
I think `minUnavailableNumber` should be `maxUnavailableNumber`.

Reply:
Thanks!